### DISCUSSION TITLE
Possibility to preserve order of files when using Make.md space

### DISCUSSION THREAD

**Discussion by Ashesh3** [KosmosisDire/obsidian-webpage-export#434] [Q&A] [answered]
Hey, I use Make.md to organize my content but when I export it, the files are not exported as they are shown on the Make.md space.

Make.md space:
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/3626859/81efefa1-0354-4b53-a52b-62998703ff64)

Exported html:
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/3626859/53cbc0a4-c78d-45dc-927f-dc20509117b4)

Is there a way to export the files exactly how they appear in my Make.md space?

**Comment by KosmosisDire** [comment, marked as answer]
No unfortunately not. Make.md icons are only supported for files not folders. And that is the only feature from make.md which is supported. File order is not a feature.

**Reply by Ashesh3** [reply]
I see, thank you for the response.

**Comment by DanielQ-CV** [comment]
is there any other way to change the file order?

**Comment by KosmosisDire** [comment]
No file order cannot be changed. In the future I plan to add the ability to change the order, but as of now there is no built in way.

**Reply by DanielQ-CV** [reply]
OK thank you for your super fast response.
Great Plugin anyway. :-)
Best regards
